wallet/src/de/schildbach/wallet/ui/WalletTransactionsFragment.java

```diff
@@ -257,18 +257,6 @@ public View onCreateView(final LayoutInflater inflater, final ViewGroup containe
         return walletTransactionsFragmentView;
     }
 
-    @Override
-    public void onResume() {
-        super.onResume();
-        //reloadTransactions();
-    }
```

Comment:
We don't need to reload the transactions onResume?  Not sure when this was commented out. or is the reload method called else where?

Reply:
@HashEngineering , we don't need. This is called from the filter. If I recall correctly I added this while making the refactoring then I was it was not needed.

Reply:
ok, great